create_compendiums.py

```diff
@@ -43,8 +44,8 @@ def informed_parse(self, filename):
         try:
             return et.parse(filename)
         except ParseError as pe:
-            print filename
-            print pe
+            print(filename)
```

Comment:
I'm, honestly, a bit surprised that changing the python version on this did not also break the travis checks, as that is specifically set up for 2.7.

I have a feeling this will come back and break something, but I'll just have to wait and see

Reply:
I did not change the Python version, I only tried to make the script compatible with both Python 2.7 and Python 3. The print statement became a function in Python 3, but that is backwards compatible, so that is why all those have been changed.

Reply:
I see, I was unaware of the backwards compatibility of its function use